mappings/net/minecraft/data/server/BlockLootTableGenerator.mapping

```diff
@@ -1 +1,67 @@
 CLASS net/minecraft/class_2430 net/minecraft/data/server/BlockLootTableGenerator
+	FIELD field_11336 NEED_SILK_TOUCH Lnet/minecraft/class_4570$class_210;
+	FIELD field_11337 DOESNT_NEED_SILK_TOUCH Lnet/minecraft/class_4570$class_210;
+	FIELD field_11338 JUNGLE_SAPLING_DROP_CHANCES_FROM_LEAVES [F
+	FIELD field_11339 SAPLING_DROP_CHANCES_FROM_LEAVES [F
+	FIELD field_11340 ALWAYS_DROPPED_FROM_EXPLOSION Ljava/util/Set;
+	FIELD field_11341 DOESNT_NEED_SILK_TOUCH_SHEARS Lnet/minecraft/class_4570$class_210;
+	FIELD field_11342 NEED_SILK_TOUCH_SHEARS Lnet/minecraft/class_4570$class_210;
+	FIELD field_11343 NEED_SHEARS Lnet/minecraft/class_4570$class_210;
```

Comment:
I think `NEEDS_*` would be better for the `NEED_*` conditions.

Reply:
Changed to `NEEDS_*`. I also changed the `REQUIRES_*` constants in `FishingLootTableGenerator` to `NEEDS_*` for consistency.